src/snapred/backend/data/LocalDataService.py

```diff
@@ -259,34 +237,6 @@ def _constructNormalizationStatePath(self, stateId, useLiteMode) -> Path:
         mode = "lite" if useLiteMode else "native"
         return self._constructCalibrationStateRoot(stateId) / mode / "normalization"
 
-    @validate_call
-    def _constructCalibrationDataPath(self, runId: str, useLiteMode: bool, version: Optional[Version]) -> Path:
-        """
-        Generates the path for an instrument state's versioned calibration files.
-        """
-        stateId, _ = self._generateStateId(runId)
-        if version is None:
-            version = self._getLatestCalibrationVersionNumber(stateId, useLiteMode)
-        return self._appendVersion(self._constructCalibrationStatePath(stateId, useLiteMode), version)
-
-    @validate_call
-    def _constructNormalizationDataPath(self, runId: str, useLiteMode: bool, version: Optional[Version]) -> Path:
-        """
-        Generates the path for an instrument state's versioned normalization calibration files.
-        """
-        stateId, _ = self._generateStateId(runId)
-        if version is None:
-            version = self._getLatestNormalizationVersionNumber(stateId, useLiteMode)
-        return self._appendVersion(self._constructNormalizationStatePath(stateId, useLiteMode), version)
-
-    @validate_call
-    def _constructCalibrationParametersFilePath(self, runId: str, useLiteMode: bool, version: Version) -> Path:
-        return self._constructCalibrationDataPath(runId, useLiteMode, version) / "CalibrationParameters.json"
-
-    @validate_call
-    def _constructNormalizationParametersFilePath(self, runId: str, useLiteMode: bool, version: Version) -> Path:
-        return self._constructNormalizationDataPath(runId, useLiteMode, version) / "NormalizationParameters.json"
-
```

Comment:
All of these methods, which were only used inside the `LocalDataService`, have been replaced by calls to `Indexor` methods.  For instance, `_constructNormalizationDataPath(runId, useLiteMode, version)` can be replaced with `normalizationIndexor(runId, useLiteMode).versionPath(version)`.